pom.xml

```diff
@@ -122,39 +127,27 @@
    </repositories>
 
    <properties>
+      <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+      <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
```

Comment:
Isn't `project.build.sourceEncoding` and `project.reporting.outputEncoding`  already be set by `spring-boot-starter-parent` ?